src/posts/faq.md

```diff
@@ -18,15 +18,12 @@ AI safety researchers (who are the experts on this topic) are divided on this qu
 Note that no (surveyed) AI safety researchers believe that there's a 0% chance.
 However, there might be selection bias here: people who work in the AI safety field are likely to do so because they believe preventing bad AI outcomes is important.
 
-If you ask AI researchers in general (not safety specialists), this number drops to a [mean value of around 14%](https://aiimpacts.org/2022-expert-survey-on-progress-in-ai/).
+If you ask AI researchers in general (not safety specialists), this number drops to a [mean value of around 14%](https://aiimpacts.org/2022-expert-survey-on-progress-in-ai/), with a median of 5%.
 A small minority, about 4% of them, believe that the alignment problem is not a real problem.
 Note that there might be a selection bias here in the opposite direction: people who work in AI are likely to do so because they believe AI will be beneficial.
 
 _Imagine you're invited to take a test flight on a new airplane_.
-The airplane safety experts on average think there's a 30% chance it would crash.
-The plane engineers think there's a 14% chance of crashing.
-About 4% of engineers are saying that there's no chance of crashing, it is uncrashable.
-
+The plane engineers think there's a 5% chance of crashing.
```

Comment:
That's not their average estimate - it's 14%. Why use the median value here?

Perhaps if you change the sentence to "the average plane engineer thinks there's a 5% chance of crashing" that makes sense.

Or "The plane engineers, on average, think there's a 14% chance of crashing".

I'd go for using the average instead of the median value.

Reply:
Good points. On manipulation I disagree, as we're not choosing the highest number, we're simply showing the average - an easier to understand, more commonly used statistical metric than the median. And we're not showing different numbers from different studies.

On pursuasiveness, you may have a point, it might convince more people if we use the lower number. But that _is_ being manipulative. 

I believe we should be honest and speak the truth. We should not choose the lower number with the less commonly used statistical tool to make our case more convincing.

Reply:
Manipulating the data to be as charitable as possible to the other side seems fine. I don't agree the average is the default choice here. I think in an opinion survey like this it is probably more standard to report the median.